app/views/cws/new.html.slim

```diff
@@ -0,0 +1,20 @@
+- content_for :page_title
+  = t(".title")
+- content_for :back do
+  = link_to t("generic.back"), estimate_path(params[:estimate_id]), class: "govuk-back-link"
+
+.govuk-grid-column-two-thirds
+  = form_for(@form, url: estimate_cws_path(params[:estimate_id])) do |form|
+    = form.govuk_error_summary t("generic.error_summary_title")
+    = form.govuk_radio_buttons_fieldset :form_type,
+                                        legend: { text: t(".title") },
```

Comment:
Hey Patrick,

Just realised the heading size is too small 

<img width="659" alt="Screenshot 2023-03-24 at 16 25 58" src="https://user-images.githubusercontent.com/76905544/227584395-b7218bf7-22b3-4eba-aca8-cd055f5a9a9a.png">

Reply:
Good spot!